### PR TITLE
Update mainClass.js

### PR DESCRIPTION
**Pull request by MrJohnCoder**

### CODE REVIEW

**Comment by iamhansen** [comment]
what in the-

File: src/mainClass.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ class Memer extends Base {
   constructor (bot) {
     super(bot)
 
-    this.bot = bot
```
Comment (iamhansen) [inline src/mainClass.js]: hmmm

Comment (iamhansen) [reply src/mainClass.js]: melmsie is probably not gonna like this

File: src/mainClass.js [hunk 2]
```diff
@@ -7,7 +7,6 @@ class Memer extends Base {
   constructor (bot) {
     super(bot)
```
Comment (iamhansen) [inline src/mainClass.js]: travis doesn't like this

File: src/mainClass.js (hunk 1 above)
Comment (iamhansen) [reply src/mainClass.js]: plus there's more than 2 lines that are blank and travis-ci doesnt like that

**Comment by Aetheryx** [comment]
> "travis doesn't like this" 

bitch travis doesn't like anything https://why-are-you-buying-clothes-at.the-soup.store/bd7d96.png
Anyhow, PR approved. We won't be merging this until after (or on) Halloween though, we're on 665 commits right now and we're gonna *try* to get the 666th commit on Halloween.

**Comment by melmsie** [comment]
can't wait.

**Comment by iamhansen** [comment]
@Aetheryx same

**Pull request merged**